websites/G/G2G/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Mattis.",
+		"id": "283757164923715587"
+	},
+	"service": "G2G",
+	"description": {
+		"en": "G2G is a world-leading digital marketplace, founded by serious technopreneurs."
+	},
+	"url": "www.g2g.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/U0FQhL3.png",
+	"thumbnail": "https://i.imgur.com/4JlsyFM.png",
+	"color": "#f03827",
+	"category": "games",
+	"tags": ["marketplace", "deals", "games"],
```

Comment:
This file is not formatted correctly. Use `yarn format` to format it.

Reply:
Hello ! Thanks for your comments, i will take care of all you said. 
I already did a yarn format ^^